docs/integrations/sources/quickbooks.md

```diff
@@ -1,12 +1,76 @@
 # QuickBooks
 
-## Overview
+This page contains the setup guide and reference information for the QuickBooks Source connector.
 
-The QuickBooks source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+## Prerequisites
 
-This source wraps the [Singer QuickBooks Tap](https://github.com/singer-io/tap-quickbooks).
+- Account at [Intuit Developer account](https://developer.intuit.com/app/developer/qbo/docs/get-started)
```

Comment:
I don't think this is correct. A Cloud user only needs an Intuit Quickbooks account. This should only be relevant for OSS setup.

Reply:
Good catch, thanks! Fixed